junqo_back/src/main.ts

```
@@ -1,8 +1,58 @@
+import { ValidationPipe } from '@nestjs/common';
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
+import { Logger } from '@nestjs/common';
+import 'dotenv/config';
+
+const PORT_MIN = 1;
+const PORT_MAX = 65535;
+const CORS_ORIGINS = process.env.CORS_ORIGINS?.split(',') || RegExp('.*');
```

Biome (1.9.4)

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)